### PR TITLE
FOOTER USING HTML and CSS

### PR DESCRIPTION
**Pull request by raghav1482**
@Rutuj-Runwal kindly review my PR.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -196,11 +199,29 @@ <h2 style="text-align:center">Don't contribute to these phishing statistics!</h2
...
+            <li>Decreases Bandwidth Consumtion</li>
+            <li>Improves Privacy</li></ul></p>
+          <img src="img\RR_logo.png" style="width:10%;float: right;position: relative;top:-13vw;right: 50px;">
         </div>
-    </footer>
+        <hr width="50%" style="margin:0px 0px 30px 27vw;">
+      <div class="footer-bottom">
+          <p>Copyright © All Rights Reserved | Developed by &nbsp;<a href="https://bit.ly/let_us_connect">Rutuj Runwal</a>  </p>
+                  <div class="footer-menu">
+                    <ul class="f-menu">
```
Comment (Rutuj-Runwal) [inline index.html]: Why are you recreating entire navbar inside the footer?
Unnecessary

**Comment by raghav1482** [comment]
@Rutuj-Runwal kindly review my PR #15

**Comment by Rutuj-Runwal** [comment]
> @Rutuj-Runwal kindly review my PR #15 

I already added comments
Are they not visible to you?

**Comment by raghav1482** [comment]
No sir....

**Comment by raghav1482** [comment]
> > @Rutuj-Runwal kindly review my PR #15
> 
> I already added comments Are they not visible to you?

no sir

**Comment by Rutuj-Runwal** [comment]
> > > @Rutuj-Runwal kindly review my PR #15
> > 
> > I already added comments Are they not visible to you?
> 
> no sir



I think some issue with GitHub here take a look👆

**Review by Rutuj-Runwal — CHANGES_REQUESTED** [review]
This type of content already exists in the page and has no real use in the footer

**Comment by raghav1482** [comment]
> > > > @Rutuj-Runwal kindly review my PR #15
> > > 
> > > 
> > > I already added comments Are they not visible to you?
> > 
> > 
> > no sir
> 
> 
> 
> I think some issue with GitHub here take a look👆

Sir i have created the navbar to give the brief link of contents above.
Easy navigation from bottom.

**Review by raghav1482 — COMMENTED** [review]
@Rutuj-Runwal sir i have made the changes plzz review the same

**Comment by Rutuj-Runwal** [comment]
> @Rutuj-Runwal sir i have made the changes plzz review the same

Don't call me sir, we are all here to learn🙂

**Comment by Rutuj-Runwal** [comment]
If you check the website,
The line no 204 to 210 that you have added already exists in earlier content and it is redundant
Also you have removed the font awesome heart favicon that was present earlier
Check the footer at : https://rutuj-runwal.github.io/RRAdblocker/

**Comment by raghav1482** [comment]
> 
@Rutuj-Runwal  okk big brother.

**Pull request closed without merging**